corehq/apps/export/models/new.py

```diff
@@ -1779,6 +1782,7 @@ def generate_schema_from_builds(
             be present in the schema since past builds have not been
             processed.
         :param task: A celery task to update the progress of the build
+        :param for_new_export_instance: Flag to be set if generating schema for a new export instance
```

Comment:
Ya, I was thinking of that originally. Though that assumption will not always hold.
Like for this use of this function, https://github.com/dimagi/commcare-hq/blob/902a3de50f0538a8a5d84c9e11f632b1b6711a9f/corehq/apps/export/models/new.py#L1841, a fresh schema is being passed to it. Hence, I kept it really explicit so that we only do the reordering when needed.